doc/modules/decomposition.rst

```diff
@@ -861,19 +861,50 @@ discovering abstract topics from a collection of documents.
 
 The graphical model of LDA is a three-level Bayesian model:
 
-.. image:: ../images/lda_model_graph.png
```

Comment:
It looks like lda_model_graph.png isn't used anymore so we can remove it, or maybe better rename lda_model_graph2.png into lda_model_graph.png

Reply:
Done in commit 2a94b30